constants/members.json

```diff
@@ -3188,30 +3196,30 @@
     {
       "name": "Cindy Li",
       "role": "vp internal",
-      "term": 1235,
+      "term": 1236,
       "teams": ["executive"],
-      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fs23headshots%2FVPI_Cindy_Li.jpg?alt=media&token=0ea951b4-629e-4615-a5b4-19d22cbc5280&_gl=1*wajfif*_ga*MTUwMDEyNjY3My4xNjc0MDgyODE2*_ga_CW55HF8NVT*MTY4NTkwMDY2Mi4zLjEuMTY4NTkwMTk1NS4wLjAuMA.."
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVPI_Cindy_Li.jpg?alt=media&token=35611a3e-ea9b-4bb1-a2a8-963d9ac954c4"
     },
     {
       "name": "Shirley Fang",
```

Comment:
She doesn't seem to be on the list for executive list. Ik she is on the pics list, but let's leave her `term` to `1235` for now